packages/reporter/src/lib/app-state.ts

```diff
@@ -22,6 +22,7 @@ const defaults: DefaultAppState = {
 }
 
 class AppState {
+  @observable autoScrollingEnabledUnderUserPreferences = true
```

Comment:
lol I am not a fan of this name, would almost rather see this continue to be `autoScrollingEnabled` and add a new one for the "temp" auto scrolling as like `isAutoscrolling` or something. 🤷🏻‍♀️ but not going hold this back

Reply:
I struggled to name here, since `autoScrollingEnabled` encompasses really any type of explicit/implicit mix. We already have the idea of "temporary" autoscroll, but that just sets `autoScrollingEnabled` without a reset value. So this variable almost functions as a super set, where it makes the behavior clearer, where the idea of temp auto-scrolling isn't necessarily temporary and might be whatever behavior is explicitly/implicitly set. 

All to say I don't really love this name 😂

Reply:
how about `autoScrollingUserPref`? Not sure why enabled is in there at all since it's a bool and it doesn't need to be a sentence.

Reply:
@emilyrohrbough @mjhenkes renamed in aebfa73196ce0ab018c34c4245109340df3fcdbb